geoextent/lib/extent.py

```diff
@@ -121,21 +121,17 @@ def fromFile(filePath, bbox=True, tbox=True, num_sample=None):
 
     # get the module that will be called (depending on the format of the file)
 
-    usedModule = get_module(filePath)
+    for i in handle_modules:
+        valid = handle_modules[i].checkFileValidity(filePath)
```

Comment:
I like how this looks! However, the function name does not feel right anymore, does it? Isn't it more like `checkFileSupported` or so?